meshroom/core/attribute.py

```diff
@@ -152,6 +153,8 @@ def uid(self, uidIndex=-1):
     def isLink(self):
         """ Whether the attribute is a link to another attribute. """
         # note: directly use self.node.graph._edges to avoid using the property that may become invalid at some point
+        if not self._node:
+            return False
```

Comment:
Maybe: `return self._node and self.node.graph and self.isInput and self in self.node.graph._edges.keys()`
I'm not sure about the mix between self._node and self.node.